docs/tutorials/web/index.md

```diff
@@ -0,0 +1,77 @@
+
+   * Inference on server in JavaScript. Use the `onnxruntime-node` package.
+
+     Your application may have constraints that means it is better to perform inference server side.
+
+     * **The model is too large** and requires higher hardware specs. In order to do inference on the client you need to have a model that is small enough to run efficiently on less powerful hardware.
+     * You don't want the model to be downloaded onto the device.
+
+     You can also use the onnxruntime-node package in the backend of an elctron app.
+
+   * Inference on server using other language APIs. Use the `onnxruntime` packages for C/C++ and other languages.
+
+     * **If the application is performance critical**. If the onnxruntime-node server-side JavaScript library does not deliver the performance that your application needs then you can use the native ONNX Runtime libraries.
+
+   * Inference in a React Native application. Use the `onnxruntime-react-native` package.
+
+2. Which machine learning model does my application use?
+
+   You need to understand your web app's scenario and get an ONNX model that is appropriate for that scenario. For example, does the app classify images, do object detection in a video stream, summarize or predict text, or do numerical prediction.
+
+   ONNX Runtime web applications process models in ONNX format. ONNX models can be obtained from the [ONNX model zoo](https://github.com/onnx/models), converted from PyTorch or TensorFlow, and many other places.
+
+   You can also create a custom model that is specific to the task you are trying to solve. Use code to build your model or use low code/no code tools to create the model. Check out the resources below to learn about some different ways to create a customized model. All of these resources have an export to ONNX format functionality so that you can leverage this template and source code.
+
+   * [Use AutoML to create a custom model](https://docs.microsoft.com/azure/machine-learning/concept-automated-ml)
+   * [Use Custom Vision Cognitive Services to create a custom model](https://docs.microsoft.com/azure/cognitive-services/custom-vision-service/overview)
+   * [Use Azure Machine Learning Designer to create a custom model](https://docs.microsoft.com/en-us/azure/machine-learning/concept-designer)
+   * [Build your own model with PyTorch.](https://docs.microsoft.com/learn/paths/pytorch-fundamentals/)
+
+3. How do I bootstrap my app development?
+
+   Bootstrap your web application according in your web framework of choice e.g. vuejs, reactjs, angularjs.
+
+   a. Add the ONNX Runtime dependency
+   b. Consume the onnxruntime-web API in your application
+   c. Add pre and post processing appropriate to your application and model
+
+4. How do I optimize my application?
+
+   The libraries and models mentioned in the previous steps can be optimized to meet memory and processing demands.
+
```

Comment:
we can add one here:
 - tune session options ( including trying different EPs)

Reply:
Added